files/borgback-adfc-final.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+# Backup-Skript mit Prüfung auf aktive Verbindung zu den Backup-Satelliten
+# Ver. 0.7 vom 30.11.2019, Autor: Volker Kraeft [post@volkerkraeft.de]
+
+# ---------- Variablen-Definition ---------------------------------------------------------
+
+TIMESTAMP=$(date +"%Y%m%d-%H%M%S")                   # Zeitstempel für Logdatei und Backuparchiv
+LOG_DIR="/pfad/zum/logverz"                          # Verzeichnis für Logdaten
```

Comment:
Bitte das /pfard/zum/ ... durch tatsächliche Werte ersetzen, das würde ich auch gerne reviewen, wo wir das speichern wollen.

Reply:
Ich hatte in meinem LAN mit meinen dort gültigen Pfaden getestet. Da diese anderen vermutlich wenig sagen, habe ich in der vorläufigen Endfassung /sprechende/pfad/angaben verwendet. Diese können für den Produktivbetrieb sinnvoll und nach Belieben ersetzt werden.

Reply:
Wir sollten die aber in  diesem Code-Review reviewen können, damit wir uns einig sind, wo was hin soll.